app/views/mail/newAdminWKOrg.scala.html

```diff
@@ -1,25 +1,25 @@
-@( name: String )
+@(name: String, demoDatasetUrl: String)
 
 @emailBaseTemplate() {
   <p>Hi @name,</p>
   <p>
-    welcome to your new webKnossos account. You are ready to start exploring and to collaboratively
-    annotate large 3D datasets.
+    welcome to your new webKnossos account. You are ready to share and annotate your large 3D datasets.
   </p>
   <h2 style="font-size: 1rem;">Get started</h2>
   <ul style="font-size: 1rem;">
     <li>
-      Use webKnossos for free with any of the public datasets on
-      <a href="https://webknossos.org">webknossos.org</a>. All webKnossos features can be used
-      with your new free account.
+      <a href="@demoDatasetUrl">Open a demo dataset</a> to experience webKnossos in action. 
+      There are <a href="https://webknossos.org/publications">many published datasets</a> from the scientific community ready to be explored.
     </li>
     <li>
-      Upload your own dataset into your own space private workspace. If desired, you can
-      publish your dataset into our public gallery.
+      <a href="https://webknossos.org/datasets/upload">Upload your own dataset</a>. Datasets are private by default and you can securely share them with your collaborators.
     </li>
     <li>
-      Invite colleagues from your lab or external collaborators to work with your data.
-      webKnossos works great for small teams and supports management of large annotator crowds.
+      Watch a short video to <a href="https://webknossos.org/datasets/upload">learn how to create annotations</a> in webKnossos.
```

Comment:
This URL should probably point to a youtube vide?

Reply:
I added the youtube-link that is also used in the welcome-header. Should we put the url in the application.conf? cc @fm3

Reply:
Since this URL doesn't need to be instance specific, I'd be fine with not having it in the application.conf. Updating it fast is also not a big priority imo. But either way would be fine with me.

Reply:
The welcome header/onboarding uses three different video URIs. Not sure how to (/if we should) move them all to config.

Reply:
ok. then let's keep it hard-coded.